account_parent/models/account.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 Steigend IT Solutions
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import odoo.addons.decimal_precision as dp
+from odoo import api, fields, models
+
+
+class AccountAccountType(models.Model):
+    _inherit = "account.account.type"
+
+    type = fields.Selection(selection_add=[('view', 'View')])
+
+
+class AccountAccount(models.Model):
+    _inherit = "account.account"
+
+    @api.model
+    def _move_domain_get(self):
+        context = dict(self._context or {})
```

Comment:
I think this function should return 2 values: 
 - domain
 - params

Then you can build safely the SQL query avoiding SQL injection.